ui/lib/core/addon/templates/components/replication-header.hbs

```diff
@@ -15,7 +15,7 @@
 {{/if}}
 <PageHeader as |p|>
   <p.top>
-    {{#if (not isSummaryDashboard) }}
+    {{#if (not (or isSummaryDashboard isSecondary)) }}
```

Comment:
If is drSecondary then don't show breadcrumb, there is a conditional higher up that is making sure this is DR.